Gordon360/Services/HousingService.cs

```diff
@@ -606,25 +591,16 @@ public ApartmentApplicationViewModel GetApartmentApplication(int applicationID)
                 // We will have to decide what is the best course of action in this case
             }
 
-            GET_AA_APPLICATIONS_BY_ID_Result applicationsDBModel = applicationResult.FirstOrDefault(x => x.AprtAppID == applicationID);
+            GET_AA_APPLICATIONS_BY_ID_Result applicationDBModel = applicationResult.FirstOrDefault(x => x.AprtAppID == applicationID);
 
-           StudentProfileViewModel editorStudent = Data.StudentData.FirstOrDefault(x => x.AD_Username.ToLower() == applicationsDBModel.EditorUsername.ToLower());
-            if (editorStudent == null)
+            // Assign the values from the database to the custom view model for the frontend
+            ApartmentApplicationViewModel apartmentApplicationModel = applicationDBModel; //implicit conversion
```

Comment:
Oh, this is the reason you added that extra section in the view model definition, isn't it?

Reply:
Yes, implicit conversions wrap all that convoluted conversion stuff into the definition, so that it can easily be reused in one-liners like this.
We need these conversions so frequently and they are always the same mapping of variables, so we can avoid repeating the code in place